sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -36,6 +37,16 @@ public class AzureMonitorOptions : ClientOptions
         /// </summary>
         public bool DisableOfflineStorage { get; set; }
 
+        /// <summary>
+        /// Enables or disables the Live Metrics feature. This property is enabled by default.
+        /// Note: Enabling Live Metrics incurs no additional billing or costs. However, it does introduce
+        /// a performance overhead due to extra data collection, processing, and networking calls. This overhead
+        /// is only significant when the LiveMetrics portal is actively used in the UI. Once the portal is closed,
+        /// LiveMetrics reverts to a 'silent' mode with minimal to no overhead.
+        /// <see href="https://learn.microsoft.com/azure/azure-monitor/app/live-stream?tabs=dotnet6"/>.
+        /// </summary>
+        public bool EnableLiveMetrics { get; set; } = true;
```

Comment:
https://learn.microsoft.com/en-us/azure/azure-monitor/app/live-stream?tabs=dotnet6#unexpected-large-number-of-requests-to-livediagnosticsmonitorazurecom

If both AI SDK and OTel are present, they will track each other's livemetrics call (along with normal ingestion calls too), which will cause billing surprise. Should we get a doc place to warn about the known issues when using classic ai sdk and otel in same app?

Reply:
Yes, we should cover in our migration doc. Mothra will start the work in couple of weeks.